packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -152,3 +186,66 @@ async def publish(self, exchange_name: str, message: Message) -> None:
                 aio_pika.Message(message.encode()),
                 routing_key="",
             )
+
+    async def rpc_request(
+        self,
+        namespace: RPCNamespace,
+        method_name: str,
+        *,
+        timeout: Optional[PositiveInt] = 5,
+        **kwargs: dict[str, Any],
+    ) -> Any:
+        """
+        Call a remote registered `handler` by providing it's `namespace`, `method_name`
+        and list of expected arguments.
+
+        NOTE: `namespace` should always be composed via `get_namespace`
+        """
+
+        if not self._rpc:
+            raise RPCNotInitializedError()
+
+        namespaced_method_name = _get_namespaced_method_name(namespace, method_name)
+        try:
+            queue_expiration_timeout = timeout
+            awaitable = self._rpc.call(
+                namespaced_method_name,
+                expiration=queue_expiration_timeout,
+                kwargs=kwargs,
+            )
+            return await asyncio.wait_for(awaitable, timeout=timeout)
+        except MessageProcessError as e:
+            if e.args[0] == "Message has been returned":
```

Comment:
Is there not a better way to map this but that with a error message??
If there is no other way, is there perhaps a constant provided by aiopicka?

Reply:
Actually I was annoyed about it.
The reason is pretty obscure and hard to figure out.
There was no constant or a better message. But there are tests around this to ensure the interface does not change since we rely on it.

Reply:
question, did you try this?
```python
rpc = await RPC.create(channel, host_exceptions=True)
```
coming from https://aio-pika.readthedocs.io/en/latest/apidoc.html#aio_pika.patterns.RPC